tests/unittests/test_updated_bookmark.py

```diff
@@ -37,3 +37,53 @@ def test_update_bookmark(self, mock_now, mock_get_query_params, mock_update_book
         customers = list(CUSTOMER_OBJECT.get_objects())
         # Verify that the min-max evaluation returns correct results and provided in the update_bookmark()
         mock_update_bookmark.assert_called_with(strftime(strptime_to_utc(NOW_TIME) - datetime.timedelta(DATE_WINDOW_SIZE)))
+
+
+    @mock.patch("tap_shopify.streams.base.Stream.call_api")
+    @mock.patch("tap_shopify.streams.base.Stream.get_bookmark", return_value=strptime_to_utc('2021-08-11T01:55:05-04:00'))
+    @mock.patch("tap_shopify.streams.base.Stream.update_bookmark")
+    @mock.patch("tap_shopify.streams.base.Stream.get_query_params")
+    @mock.patch("singer.utils.now", return_value=strptime_to_utc(NOW_TIME))
+    def test_since_id_and_updated_at_max_deleted(self, mock_now, mock_get_query_params, mock_update_bookmark, mock_get_bookmark, mock_call_api):
+        """Verify after successful sync since_id and updated_at_max keys are deleted from the state"""
+        mock_call_api.return_value = [CUSTOMER_1, CUSTOMER_2]
```

Comment:
Why is this return value set here, but some of the other patches are set above?

Reply:
Pattern was duplicated from the existing test for conistency, will refactor the code at both the places.

Reply:
My opinion on consistency is only copy it when it's good. Otherwise, we should improve the code when we touch it